std/experimental/normint.d

```diff
@@ -0,0 +1,571 @@
+// Written in the D programming language.
+
+/**
+This module implements support for normalized integers.
+
+Authors:    Manu Evans
+Copyright:  Copyright (c) 2015, Manu Evans.
+License:    $(WEB boost.org/LICENSE_1_0.txt, Boost License 1.0)
+Source:     $(PHOBOSSRC std/experimental/normint.d)
+*/
+module std.experimental.normint;
```

Comment:
Indeed. I'll probably start a newsgroup thread about this some time. OTOH module renaming/refactoring is usually not a popular idea in the D newsgroup ;-)